.github/ISSUE_TEMPLATE/operate-feature.md

```diff
@@ -6,38 +6,33 @@ labels: ["kind/feature", "component/operate"]
 assignees: ''
 
 ---
-<!-- Remember to link this feature issue in the parent Epic -->
 
-### Description
-<!-- [Optional] A clear description of the story -->
+### Description (required on creation)
```

Comment:
In the past I often left out a description in favor of a user story that describes the feature sufficiently. Is it okay if we keep this optional in case there is user story?

Reply:
I'd also be fine to remove the description entirely. In the C7 template we don't have it: https://github.com/camunda/camunda-bpm-platform/blob/master/.github/ISSUE_TEMPLATE/feature_request.md
I left it in for now because it was there before, so I thought people like to provide additional information to the user story. But in my opinion not needed.

Reply:
I'm removing the description section. We can bring it back if someone misses it.